# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/moz.build
# Commit: 7707637480e7
# Full Hash: 7707637480e7a5efa16c33cee89f10f5ce6be48f
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2023-02-07 04:35:34
# Regressor Bug: 1799258
# File Overlap Count: 1
# Description:
#   Bug 1799258 - Share all-of-dcomp.h preamble, and deal with outdated mingw dcomp.h. r=gfx-reviewers,sotaro
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D168839
# ==============================================================================

diff -r 0141b29bf5df -r 7707637480e7 gfx/thebes/moz.build
--- a/gfx/thebes/moz.build	Mon Feb 06 19:58:55 2023 +0000
+++ b/gfx/thebes/moz.build	Mon Feb 06 19:58:56 2023 +0000
@@ -67,6 +67,7 @@
 ]
 
 EXPORTS.mozilla.gfx += [
+    "AllOfDcomp.h",
     "D3D11Checks.h",
     "DeviceManagerDx.h",
     "DisplayConfigWindows.h",